Make edits to the demo data section (#3020)
influxdata/docs-v2: content/influxdb/cloud/get-started.md

influxdb/cloud/tags: [get-started, install]
---

After you've [signed up for InfluxDB Cloud](/influxdb/cloud/sign-up/), you're ready to get started:

1. Do one of the following:
   - Add [demo data](#add-demo-data).
   - Add [sample data](#add-sample-data). 

>    - Use your own data to explore Cloud. See how to [write your data](/influxdb/cloud/write-data/) to Cloud.

Reviewer: ```suggestion
   - [Use your own data](/influxdb/cloud/write-data/) to explore InfluxDB Cloud.
```

> 2. [Create a notebook](#create-a-notebook) by clicking **Books > Create Notebook**, and then do the following: 
>      1. [Select metrics](#select-metrics)
>      2. [Visualize data](#visualize-data)
>      3. (Optional) [Process data](#process-data)
>      4. (Optional) [Monitor data](#monitor-data)
>      5. (Optional) [Output to a new bucket and export as a task](#output-to-a-new-bucket-and-export-as-a-task)

Reviewer: ```suggestion
2. [Create a notebook](#create-a-notebook):
    1.  Clicking **Books > Create Notebook** in the navigation menu on the left.
        {{< nav-icon "books" >}}
    2. [Select metrics](#select-metrics)
    3. [Visualize data](#visualize-data)
    4. (Optional) [Process data](#process-data)
    5. (Optional) [Monitor data](#monitor-data)
    6. (Optional) [Output to a new bucket and export as a task](#output-to-a-new-bucket-and-export-as-a-task)
```

## Add demo data

This example uses InfluxDB Cloud's Website Monitoring demo data bucket. To add the demo data bucket:

1.  Click **Data (Load Data)** > **Buckets** in the navigation menu on the left.

    {{< nav-icon "data" >}}

2.  Click **{{< icon "plus" >}} Add Demo Data**, and then select the **Website Monitoring** bucket.
3.  The Demo Data bucket appears in your list of buckets.

## Add sample data

{{% note %}}

> For example purposes, we will be referencing the [NOAA water sample data](/influxdb/v2.0/reference/sample-data/#noaa-water-sample-data), but we provide other [sample data sets](/influxdb/cloud/reference/sample-data/#sample-datasets) as well.

Reviewer: ```suggestion
The examples below use the [NOAA NDBC sample data](/influxdb/v2.0/reference/sample-data/#noaa-ndbc-data), but we provide other [sample data sets](/influxdb/cloud/reference/sample-data/#sample-datasets) as well.
```

{{% /note %}}

Use [sample data](/influxdb/cloud/reference/sample-data/) to quickly populate InfluxDB with sample time series data. Sample data sets delivered with Cloud are used throughout this documentation site to provide context, best practices, and clear examples for how InfluxDB Cloud features and functions work.

1. Click **Data > Bucket** and click **Create Bucket** to create a bucket and name it **noaa**.
2. Do one of the following to download sample data: 
   - [Install community templates](#add-sample-data-with-community-templates) 

>    - [Download using the InfluxData UI](#add-sample-data-with-influxdata-ui)

Reviewer: ```suggestion
  - [Write sample data with an InfluxDB task](#write-sample-data-with-an-influxdb-task)
```

> ### Add sample data with community templates 
> 
> After installing the template, sample data will be stored in the sample data bucket. 
> 
> 1. Go to the community templates respository on GitHub and head to our [sample data documentation](/influxdata/community-templates/tree/master/sample-data). 
> 2. [Install the template](https://github.com/influxdata/community-templates/tree/master/sample-data#quick-install) with the instructions given. 
> 

Reviewer: ```suggestion
### Add sample data with community templates 

1. Visit the **InfluxDB templates page** in the InfluxDB Cloud UI. Click **Settings** > **Templates** in the navigation menu on the left.

    {{< nav-icon "settings" >}}
      
2. Paste the Sample Data community temple URL in **resource manifest file** field:

    ```
    https://github.com/influxdata/community-templates/blob/master/sample-data/sample-data.yml
    ```

The sample data template installs a task that collects sample data and dashboards to visualize the sample data.
After installing the template, sample data is downloaded and stored in the sample data bucket at regular intervals.
```

> ### Add sample data in InfluxDB Cloud UI 

Reviewer: ```suggestion
### Write sample data with an InfluxDB task
```

> 1. Click **Data Explorer** in InfluxDB Cloud's left navigation menu and select your **noaa** bucket. 
> 2. Click **Script Editor**. 
> 3. Copy the `sample.data()` function listed for the NOAA sample dataset and paste it into the script editor. 
>     ```js
>     import "influxdata/influxdb/sample"
> 
>     option task = {
>       name: "Collect NOAA NDBC data"
>       every: 15m,
>     }
> 
>     sample.data(set: "noaa")
>       |> to(
>           org: "example-org",
>           bucket: "example-bucket"
>       )
>     ```
>    Click here for more information about [NOAA water sample data](/influxdb/v2.0/reference/sample-data/#noaa-water-sample-data), or other [sample data sets](/influxdb/v2.0/reference/sample-data/). 
> 4. Click **Submit**. 

Reviewer: ```suggestion
Use the [Flux InfluxDB sample package](/{{< latest "flux" >}}/stdlib/influxdata/influxdb/sample/) to download and write sample data to InfluxDB.
Add the following as an [InfluxDB task](/influxdb/cloud/process-data/manage-tasks/create-task/).

```js
import "influxdata/influxdb/sample"

option task = {
  name: "Collect NOAA NDBC data"
  every: 15m,
}

sample.data(set: "noaa")
  |> to(bucket: "noaa"  )
    ```
For more information about this and other InfluxDB sample datasets, see [InfluxDB sample data](/influxdb/cloud/reference/sample-data/). 
```

By default, a visualization cell appears below your initial metric selector cell showing the mean values on a graph. 

1. Use the function dropdown menu to view different functions to apply to your data. By default, the visualization is set to the mean function.
2. Use the visualization type dropdown to view different visualization types available.
For a complete list of options, see [Visualization types](/influxdb/cloud/visualize-data/visualization-types/).
3. Click the gear icon ({{< icon "gear" >}}) to edit specific settings for the visualization type, such as colors, orientation, and labels.

### Process data

A common processing task is downsampling data to view it at different levels of granularity and reduce overall data disk usage over time. The following steps use a Flux script to downsample your data to every 3 hours 30 seconds.

For more information on how to process data, see [here](/influxdb/cloud/process-data/).

1. Click **+** to add a cell, then select **Transform > Downsample**.
2. In the **Apply aggregate** field, select **mean**.

> 3. In the **Every Window Period** field, enter a value. For this example, we'll use `3h30s`, but any duration is supported by InfluxDB Cloud. For more information, see [duration literals](/influxdb/v2.0/reference/flux/language/lexical-elements/#duration-literals).

Reviewer: ```suggestion
3. In the **Every Window Period** field, enter a value. For this example, we'll use `3h30s`, but any duration is supported by InfluxDB Cloud. For more information, see [duration literals](/{{< latest "flux" >}}/spec/lexical-elements/#duration-literals).
```

4. Click **Preview** to view the downsampled data.

### Monitor data

Get notifications every time your data crosses a threshold. Specify your alerts by time and decide how the information will be sent to you. 

> 1. Click **+** to add a cell, then select **Output > Alert**.

Reviewer: ```suggestion
1. In your notebook, click **{{< icon "plus" >}}** to add a cell, then select **Output > Alert**.
```

After you've [signed up for InfluxDB Cloud](/influxdb/cloud/sign-up/), you're ready to start writing, querying, and visualizing data.

To get started working in Cloud, we recommend using notebooks to explore demo data and sample data. 

> To write data into InfluxDB, you need the following:
>  
> - **organization** – _See [View organizations](/influxdb/cloud/organizations/view-orgs/#view-your-organization-id)
>  for instructions on viewing your organization ID._
> - **bucket** – _See [View buckets](/influxdb/cloud/organizations/buckets/view-buckets/) for
>  instructions on viewing your bucket ID._
> - **API token** – _See [View tokens](/influxdb/cloud/security/tokens/view-tokens/)
>  for instructions on viewing your API token._
> - **InfluxDB Cloud region URL** – _See [InfluxDB Cloud regions](/influxdb/cloud/reference/regions/)_.

Reviewer 2: ```suggestion
```
